### PR TITLE
ARROW-15001: [Website] Update committers

### PR DESCRIPTION
**Pull request by bkmgit**

### CODE REVIEW

File: _data/committers.yml [hunk 1]
```diff
@@ -47,6 +47,14 @@
   role: PMC
   alias: apitrou
   affiliation: Independent / Ursa Computing
+- name: Ben Kietzman
+  role: PMC
+  alias: bkietz
+  affiliation: Ursa Computing
```
Comment (jorisvandenbossche) [inline _data/committers.yml:53]: A bit a separate issue, but all "Ursa Computing" entries should actually be changed to "Voltron Data"

Comment (bkmgit) [reply _data/committers.yml]: Updated

Comment (jorisvandenbossche) [reply _data/committers.yml]: Thanks!

File: _data/committers.yml [hunk 2]
```diff
@@ -46,7 +46,15 @@
 - name: Antoine Pitrou
   role: PMC
   alias: apitrou
-  affiliation: Independent / Ursa Computing
+  affiliation: Independent / Voltron Data
```
Comment (pitrou) [inline _data/committers.yml:49]:
```suggestion
  affiliation: Voltron Data
```

File: _data/committers.yml [hunk 3]
```diff
@@ -139,14 +159,14 @@
...
+  alias: todd
+  affiliation: Google
 - name: Uwe L. Korn
   role: PMC
   alias: uwe
   affiliation: Quantco
-- name: Ben Kietzman
-  role: PMC
-  alias: bkietz
-  affiliation: Ursa Computing
```
Comment (pitrou) [inline _data/committers.yml:149]: Nit, but could you add a blank line between the PMCs and the committers here?

File: scripts/check_committers.py [hunk 4]
```diff
@@ -32,7 +32,7 @@
 
 
 git_root = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
-
+git_root = os.path.join(git_root,"../")
```
Comment (pitrou) [inline scripts/check_committers.py:35]: This breaks the script for me:
```bash
$ .venv/bin/python scripts/check_committers.py 
Traceback (most recent call last):
  File "scripts/check_committers.py", line 72, in <module>
    local_roster = get_local_roster()
  File "scripts/check_committers.py", line 55, in get_local_roster
    with open(committers_yaml, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/antoine/arrow/site/../_data/committers.yml'
```

**Review by pitrou — CHANGES_REQUESTED** [review]

**Comment by pitrou** [comment]
@thisisnic @majetideepak @rdettai

File: _data/committers.yml [hunk 5]
```diff
@@ -203,10 +235,18 @@
   role: Committer
   alias: liyafan
   affiliation: Alibaba
+- name: Matthew Topol
+  role: Committer
+  alias: zeroshade
+  affiliation: FactSet Research Systems, Inc.
 - name: Neville Dipale
```
Comment (pitrou) [inline _data/committers.yml:242]: Ah, please also remove this duplicate entry for Neville. We may want to update the script to check for duplicates...

Comment (bkmgit) [reply _data/committers.yml]: Done.

File: _data/committers.yml (hunk 3 above)
Comment (bkmgit) [reply _data/committers.yml:149]: Done

File: scripts/check_committers.py (hunk 4 above)
Comment (bkmgit) [reply scripts/check_committers.py]: removed this, had assumed a workflow of
```
pushd scripts
python check_committers.py
popd
```

File: scripts/check_committers.py [hunk 6]
```diff
@@ -64,18 +74,24 @@ def get_local_roster():
         elif role == 'Committer':
             committers.add(uid)
         else:
-            raise ValueError(f"Invalid role {role!r} for {uid}")
+             raise ValueError(f"Invalid role {role!r} for {uid}")
```
Comment (pitrou) [inline scripts/check_committers.py:77]: Is this change deliberate?

File: scripts/check_committers.py [hunk 7]
```diff
@@ -64,18 +74,24 @@ def get_local_roster():
...
     return Roster(committers, pmcs)
 
-
 if __name__ == "__main__":
+    with open(committers_yaml, "r") as f:
+        d = yaml.safe_load(f)
```
Comment (pitrou) [inline scripts/check_committers.py:82]: It doesn't seem this variable is used, is it?

**Review by pitrou — COMMENTED** [review]
Just two small comments, thank you.

File: scripts/check_committers.py (hunk 6 above)
Comment (bkmgit) [reply scripts/check_committers.py]: No. Removed.

File: scripts/check_committers.py (hunk 7 above)
Comment (bkmgit) [reply scripts/check_committers.py]: Thanks. Removed.

**Review by pitrou — APPROVED** [review]

**Review by pitrou — APPROVED** [review]

**Comment by pitrou** [comment]
@nealrichardson Any comments on this?

File: _data/committers.yml [hunk 8]
```diff
@@ -139,14 +159,15 @@
   role: PMC
   alias: tdunning
   affiliation: MapR
+- name: Todd Lipcon
```
Comment (toddlipcon) [inline _data/committers.yml:162]: FYI I think I'm considered "emeritus" on Arrow at the moment.

Comment (pitrou) [reply _data/committers.yml:162]: Hmm, I don't know how to query emeritus members from the Apache directory. Do we have a well-known list of emeritus Arrow members?

Comment (bkmgit) [reply _data/committers.yml:162]: One can query the repositories for activity and if no activity for a certain time period, indicate someone as inactive.  This would require updating committer list to have Jira and GitHub ids, though could be done manually at present until a policy is formulated.

Comment (pitrou) [reply _data/committers.yml:162]: I don't think we want to do that. Going "emeritus" requires an official Apache procedure: https://www.apache.org/foundation/bylaws.html#4.2

**Comment by pitrou** [comment]
I'm going to merge this. We can open a separate issue for the detection of emeritus members.

**Comment by pitrou** [comment]
Opened https://issues.apache.org/jira/browse/ARROW-15094 for detection of emeritus members.
